src/content/en/tools/workbox/v3/modules/workbox-cli.md

```diff
@@ -0,0 +1,161 @@
+project_path: /web/tools/workbox/v3/_project.yaml
+book_path: /web/tools/workbox/v3/_book.yaml
+description: The module guide for workbox-cli.
+
+{# wf_updated_on: 2017-11-27 #}
+{# wf_published_on: 2017-11-27 #}
+
+# Workbox CLI  {: .page-title }
+
+## What's the Workbox CLI?
+
+The Workbox command line interface (contained in the the
+`workbox-cli` package) consists of a Node.js program called workbox that
+can be run from a Windows, MacOS, of UNIX-compatible command line
+environment. Under the hood, workbox-cli wraps the  workbox-build module,
+and provides an easy way of integrating Workbox into a command line build
+process, with flexible configurations.
+
+## Install the CLI
+
+Installing the CLI is simple, just run the following command in your
+terminal.
+
+* {NPM}
+
+    ```
+    npm install --global workbox-cli
+    ```
+
+* {Yarn}
+
+    ```
+    yarn global add workbox-cli
+    ```
+
+## CLI Modes
+
+The CLI has four different modes:
+
+- **wizard**: A step-by-step guide to set up Workbox for your project.
+- **generateSW**: Generates a complete service worker for you.
+- **injectManifest**: Injects the assets to precache into your project.
+- **copyLibraries**: Copy the Workbox libraries into a directory.
+
+### `wizard`
+
+The Workbox wizard asks a series of questions about your local directory
+setup and which files you want precached. Your answers are used to
+generate a configuration file which can then be used when running in
+`generateSW` mode.
+
+Most developers will only need to run workbox wizard once, and you're free
+to manually customize the initial configuration file that's generated,
+using any of the supported build configuration options.
+
+To start the wizard run:
+
+```
+workbox wizard
```

Comment:
Check out this [syntax for terminal code](https://developers.google.com/internal/publishing/reference/styles#terminal_code)